tests/filter_plugins

```diff
@@ -0,0 +1 @@
+../provision/filter_plugins
```

Comment:
while invoking ansible playbooks , ansible searches for filter_plugins folder in the immediate dir tree from point of invocation .  In order to make filter_plugins available for ansible api i had to create a symlink. As far as my experience , sys.path works for python script but not ansible. However, i will try it and update you on that.